netbox/dcim/forms/model_forms.py

```diff
@@ -1577,7 +1577,7 @@ class InventoryItemForm(DeviceComponentForm):
 
     fieldsets = (
         FieldSet('device', 'parent', 'name', 'label', 'role', 'description', 'tags', name=_('Inventory Item')),
-        FieldSet('manufacturer', 'part_id', 'serial', 'asset_tag', name=_('Hardware')),
+        FieldSet('manufacturer', 'part_id', 'serial', 'asset_tag', 'status', name=_('Hardware')),
```

Comment:
`status` should be listed under the first field set as it's an operational parameter.